kayenta-stackdriver/src/test/java/com/netflix/kayenta/stackdriver/metrics/StackdriverMetricsServiceTest.java

```diff
@@ -0,0 +1,69 @@
+package com.netflix.kayenta.stackdriver.metrics;
+
+import static org.hamcrest.CoreMatchers.hasItem;
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.when;
+
+import com.google.api.services.monitoring.v3.model.ListTimeSeriesResponse;
+import com.google.api.services.monitoring.v3.model.Point;
+import com.google.api.services.monitoring.v3.model.TimeSeries;
+import com.google.api.services.monitoring.v3.model.TypedValue;
+import com.netflix.kayenta.canary.CanaryConfig;
+import com.netflix.kayenta.canary.CanaryMetricConfig;
+import com.netflix.kayenta.metrics.MetricSet;
+import com.netflix.kayenta.stackdriver.canary.StackdriverCanaryScope;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Test;
```

Comment:
Better to use the new shiny junit5 stuff.

```suggestion
import org.junit.jupiter.api.Test;
```